po/it.po

```diff
@@ -724,15 +721,16 @@ msgid "Default background"
 msgstr "Sfondo preimpostato"
 
 #: src/preferences.py:76
-#, fuzzy
 msgid "Images saving"
-msgstr "Immagini"
+msgstr "Salvataggio immagini"
 
 #: src/preferences.py:79
 msgid ""
 "JPEG and BMP images can't handle transparency. If you save your images in "
 "these formats, what do want to use to replace transparent pixels?"
 msgstr ""
+Le immagini JPEG e BMP non possono gestire la trasparenza. Se salvi le immagini in "
```

Comment:
missing quote

Reply:
i tried to edit the file myself, but it has been commit to a branch that doesn't exist and which i'm supposed to push using a button that github doesn't provide...